### PR TITLE
Feature/mzid protgroup

### PR DESCRIPTION
**Pull request by mwalzer**
@timosachsenberg @jpfeuffer 
better late than never ... was initially waiting for pr #1505 but lets open this to discuss the potential refinements for PG in mzid

### CODE REVIEW

File: src/openms/include/OpenMS/METADATA/ProteinIdentification.h [hunk 1]
```diff
@@ -232,6 +233,10 @@ namespace OpenMS
     void setSearchParameters(const SearchParameters & search_parameters);
     /// Returns the search parameters
     const SearchParameters & getSearchParameters() const;
+    /// Sets the grouping parameters
+    void setGroupingParameters(const CVTermList & grouping_parameters);
```
Comment (mwalzer) [inline src/openms/include/OpenMS/METADATA/ProteinIdentification.h]: see MzIdentMLDOMHandler.cpp line 1397

**Comment by timosachsenberg** [comment]
can you give a brief summary on the current state?

**Comment by timosachsenberg** [comment]
any update on this branch? its lurking around for quite a while now. close?

**Comment by hroest** [comment]
@mwalzer any progress here?

**Comment by mwalzer** [comment]
Was a branch to discuss how to move forward improving support for protein grouping. Too few input to invest updating.

**Pull request closed without merging**